HyperVExtension/test/HyperVExtension/HyperVExtensionTests/Services/HyperVExtensionIntegrationTest.cs

```diff
@@ -95,14 +96,14 @@ private IHost CreateTestHost()
     ///   User logged on to the VM.
     /// </summary>
     [TestMethod]
-    public void TestConfigureRequest()
+    public async Task TestConfigureRequest()

```

Comment:
Why async? This is a manual test for now. And even if we automate it later, I am not sure it can run in parallel with other tests. What if two tests try to call Configure on the same VM? Or one test doing configuration and another one trying to update DevSetupAgent. Those are not supported scenarios and these kinds of tests are not designed to run concurrently.

Reply:
I just made it async because of the IAsyncOperation, I wasn't thinking about running tests in parallel but I guess we can think about that later, unless you want it fixed in this PR